core/src/main/java/org/stellar/anchor/client/DefaultClientService.java

```diff
@@ -25,90 +19,83 @@
 @NoArgsConstructor
 public class DefaultClientService implements ClientService {
   static final Gson gson = GsonUtils.getInstance();
-  List<CustodialClientConfig> custodialClients = emptyList();
-  List<NonCustodialClientConfig> nonCustodialClients = emptyList();
+  List<CustodialClient> custodialClients = new ArrayList<>();
```

Comment:
Is there advantages to use `new ArrayList<>()` instead of `emptyList()`?

Reply:
emptyList is immutable.